getting-started/getting-started.md

```diff
@@ -0,0 +1,135 @@
+# Getting Started
+
+### Pre-requisites
+
+Operating System-specific instructions (read the applicable doc before continuing):
+
+- [Windows](windows.md)
```

Comment:
There is currently no need for special instruction pages for Linux and MacOS devices. According to the [YAGNI](https://martinfowler.com/bliki/Yagni.html), we should not include those pages. We don't even expect to need them!

We could replace this section with the following Windows specific message (which displays as an appropriate warning box for users).

```md
> [!WARNING]
> The autograder unfortunately requires special setup instructions
> in order to operate on **Windows machines**.
> 
> Read [Getting Started on Windows](windows.md) before continuing.
```

Reply:
Agreed, but I left them in case @19mdavenport or any of the other TAs have useful information to add. I only know about Windows 🤷‍♂️